themes/sensei-course-theme/templates/default/lesson.html

```diff
@@ -43,6 +43,8 @@
     <!-- wp:sensei-lms/ui {"elementClass":"sensei-course-theme__main-content","lock":{"move":false,"remove":false}} -->
     <div class="wp-block-sensei-lms-ui sensei-course-theme__main-content"><!-- wp:sensei-lms/course-theme-lesson-module /-->
 
+        <!-- wp:sensei-lms/lesson-properties { "readOnly": true, "lmTemplatePart": true } /-->
```

Comment:
Hmm not sure if we should add it to the default template. Maybe having it as an option is enough?

Reply:
Agree. Removed. 853797a599756ec32bffa7505a08953e9d0e3f51